massa-executed-ops/src/executed_denunciations.rs

```diff
@@ -153,20 +153,22 @@ impl ExecutedDenunciations {
             .denunciation_expire_periods
             .saturating_add(self.config.keep_executed_history_extra_periods);
 
```

Comment:
to avoid the clone:
```rust
let (removed, kept) = mem::take(&mut self.sorted_denunciations).into_iter().partition(...);
self.sorted_denunciations = kept;
for (_, de_indexes) in remaining {
    for de_idx in de_indexes {
      self.delete_entry(de_idx, batch);
    }
}
```

Reply:
Done